docs/content/stepbystep/IDE-setup.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'IDE Setup'
+metaTitle: 'IDE Setup | Discordeno'
+metaDescription: 'Setup VSC for discordeno'
+---
+
+# Installing
+
+There are many editors for js/ts, but I recommend Visual Studio Code as its free and fast with a lot of extensions(Including a Deno one)
```

Comment:
Add a curated list of editors that have support for Deno.
Also, IDE !== Just Editor

Reply:
Yes i know, im just simplifying